docs/source/Getting-Started/QEMU-Compile-Sources.rst

```diff
@@ -4,122 +4,65 @@ Compile Sources
 Build All Components
 ##############################################################
 
```

Comment:
nit: could you please add some notice that we've recently moved away from CMake? (so that people who return can be aligned better)